server/dex/dex.go

```diff
@@ -134,6 +134,7 @@ func newConfigResponse(cfg *DexConf, cfgAssets []msgjson.Asset, cfgMarkets []msg
 		RegFeeConfirms:   uint16(cfg.RegFeeConfirms),
 		Assets:           cfgAssets,
 		Markets:          cfgMarkets,
+		Fee:              cfg.RegFeeAmount,
```

Comment:
I didn't see any reason that we couldn't put the fee in the DEX config, rather than the `register` route result. This allows the fee to be fetched before an account is created, allowing the fee to be confirmed by the user first. If there are no objections, I'll update the spec.

Reply:
That's reasonable.